### PR TITLE
fix(e2e): broken due to customized env resources

### PR DESCRIPTION
**Pull request by iamhopaul123**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by iamhopaul123** [comment]
running e2e test now. will put down the label once it succeeds

File: e2e/internal/client/cli.go [hunk 1]
```diff
@@ -284,10 +285,15 @@ copilot env init
...
 func (cli *CLI) EnvInit(opts *EnvInitRequest) (string, error) {
+	useDefault := "true"
+	if opts.CustomizedEnv {
+		useDefault = "false"
+	}
 	commands := []string{"env", "init",
 		"--name", opts.EnvName,
 		"--app", opts.AppName,
 		"--profile", opts.Profile,
+		"--no-custom-resources", useDefault,
```
Comment (efekarakus) [inline e2e/internal/client/cli.go:296]: nit: 
```suggestion
		"--no-custom-resources", !opts.CustomizedEnv,
```

Comment (bvtujo) [reply e2e/internal/client/cli.go:296]: I know this is out of scope but do we need `--no-custom-resources` to take an argument at all here? Couldn't it just be a boolean flag, and not require customers to type "true" or "false"? Or am I misunderstanding how the e2es work?

Comment (iamhopaul123) [reply e2e/internal/client/cli.go]: it doesn't need an argument... it's just by taking an argument can make it easier in e2e test

Comment (iamhopaul123) [reply e2e/internal/client/cli.go]: addressed

**Comment by efekarakus** [comment]
We can do David's flag rename as a follow up

**Pull request merged**